providers/common/kdfs/tls1_prf.c

```diff
@@ -276,23 +310,31 @@ static int tls1_prf_P_hash(const EVP_MD *md,
     unsigned char Ai[EVP_MAX_MD_SIZE];
     size_t Ai_len;
     int ret = 0;
-    OSSL_PARAM params[4];
+    OSSL_PARAM params[10], *p = params;
```

Comment:
[6] is required here, not [10]

Reply:
Yeah, I set 10 to be safe before i started modifying and forgot to adjust.  Will fix.